WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartRepository.kt

```diff
@@ -74,14 +78,14 @@ class QuickStartRepository
     override fun buildSource(siteId: Int) = flow {
         emit(QuickStartUpdate())
         refresh.asFlow().map {
-            if (quickStartUtils.isQuickStartInProgress(siteId)) {
+            if (quickStartUtils.isQuickStartInProgress(siteId) || true) {
```

Comment:
This seem to have been left here by mistake?

```suggestion
            if (quickStartUtils.isQuickStartInProgress(siteId)) {
```

Reply:
yeah, good catch! I use it for testing :-)